pkg/resources/management.cattle.io/v3/setting/validator_test.go

```diff
@@ -147,15 +129,15 @@ func (s *SettingSuite) validate(op v1.Operation) {
 
 	for _, test := range retentionTests {
 		test := test
-		s.Run(test.name(), func() {
-			t := s.T()
+		t := s.T()
+		t.Run(test.name(), func(t *testing.T) {
 			t.Parallel()
 
 			oldObjRaw, err := test.toOldSetting()
-			assert.NoError(t, err, "failed to marshal old Setting")
+			require.NoError(t, err, "failed to marshal old Setting")
```

Comment:
Nit: If we are going to swap to require here then we probably should swap to it below as well, not sure we get value out of mixing the two.

Reply:
I don't think I'd agree. There is no point in proceeding if we failed to marshal settings - thus switching to `require` regardless of how we assert the result of the `Admit()` call, which leverages the fact that `assert.X()` returns the `bool` indicating if it's succeeded (unlike require).